FrontEnd/src/components/adminPage/detail-view/DetailCheckbox.module.css

```diff
@@ -0,0 +1,59 @@
+.user-detail-checkbox {
```

Comment:
Since it seems to be shared component we can make naming less specific

Reply:
Done